coalib/processes/Processing.py

```diff
@@ -312,6 +312,11 @@ def instantiate_bears(section,
     :param console_printer:  Object to print messages on the console.
     :return:                 The local and global bear instance lists.
     """
+    profile = (False if 'profile' not in section or (
+                section['profile'].value == 'False') else (
```

Comment:
and what if it's `false`? ;) This is not so bullet proof, especially as coala allows words like "yes", "nope" ... as boolean values. I know we have done it for the debugger feature like this, but maybe now is the time to start thinking about that.

Reply:
[FALSE_STRINGS](https://github.com/coala/coala/blob/8ffd2237f84ddaca50327a89ce0132dc9999357f/coalib/misc/Constants.py#L71) doesn't included "false" ?